libs/l2/tests/test_l2.py

```
@@ -0,0 +1,5 @@
+from l2 import hello
+
+def test_hello() -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)